### PR TITLE
Improve Nav Search

### PR DESCRIPTION
**Pull request by spbyrne**
- Adds working excerpt to search results
- Hides excerpts for title hits
- Reduce excerpt generation from 3000 to 200 characters
- Strip markdown from excerpts
- Improve hit components

Todo: fix issue with out of date index data (@dwalkr  is on it)

### CODE REVIEW

File: components/search/hitComps.tsx [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
+  </DynamicLink>
 )
 
-const BlogHit = (clickHandler: any) => ({ hit }: { hit: Hit }) => (
-  <div>
-    <span onClick={clickHandler}>
-      <DynamicLink href={path.join('/blog', (hit as any).slug)}>
-        <h4 onClick={clickHandler}>
+const BlogHit = (clickHandler: any) => ({ hit }: { hit: Hit }) => {
+  console.log(hit)
```
Comment (kendallstrautman) [inline components/search/hitComps.tsx:22]:
```suggestion
```

**Comment by kendallstrautman** [comment]
Since the index data is out of date, I'm assuming I can't see / test the reformatted excerpts?

**Comment by spbyrne** [comment]
@kendallstrautman  You can test/see the new excerpts, it's just some old excerpts show up in cases where the page was moved or removed. Not all old data is kept, just in cases where the incoming data doesn't match up with a slug (I think?).

**Comment by kendallstrautman** [comment]
Weird, yeah I pulled it down and all the excerpts seem long / with markup

**Comment by spbyrne** [comment]
I bet something else trigged a regen of the indices. They're stored externally (algolia). I can trigger a regen right now and see if that fixes the preview.

**Comment by dwalkr** [comment]
> I bet something else trigged a regen of the indices. They're stored externally (algolia). I can trigger a regen right now and see if that fixes the preview.

until this is merged, any push to master will rebuild the indices with the old rules.

**Pull request merged**